contracts/stabilizer/TribeReserveStabilizer.sol

```diff
@@ -55,7 +59,9 @@ contract TribeReserveStabilizer is ITribeReserveStabilizer, ReserveStabilizer {
     /// @notice exchange FEI for minted TRIBE
     /// @dev Collateralization oracle price must be below threshold
     function exchangeFei(uint256 feiAmount) public override returns(uint256) {
-        require(isCollateralizationBelowThreshold(), "TribeReserveStabilizer: Collateralization ratio above threshold");
+        // the timer counts down from first time below threshold
+        // if oracle remains below for entire window, then begin opening exchange
+        require(isTimeEnded(), "TribeReserveStabilizer: Oracle delay");
```

Comment:
you can also use the `afterTime` modifier I think, but it's clear this way with the comment 👍

Reply:
Oooh, yeah I like afterTime